lib/i18n/payment-method-provider-type.js

```diff
@@ -28,6 +29,10 @@ const i18nTypes = defineMessages({
     id: 'Crypto',
     defaultMessage: 'Crypto',
   },
+  [TransactionKind.ADDED_FUNDS]: {
+    id: 'Added Funds',
```

Comment:
We already have a string for this one, let's reuse it

```suggestion
    id: 'Transaction.kind.ADDED_FUNDS',
```

Reply:
Yes I saw that inside lang files and I was unsure about this part of my change. Thanks for suggesting!